actions/qemuimg2disk/v1/Makefile

```diff
@@ -0,0 +1,3 @@
+export DOCKER_BUILDKIT=1
+image:
+	docker build -t nshalman/qemuimg2disk:1.0 .
```

Comment:
Is there automatic building and pushing of action containers, or should the makefile be updated with a target to build and push? What should the name be?

Reply:
I copied  and modified the Makefile from the kexec action. Hopefully that's correct.